ps2.xml

```diff
@@ -1182,6 +1182,1226 @@
+                                    <name>RC1</name>
+                                    <description>Alpha value from Read Circuit 1 for output selection.</description>
+                                    <value>0</value>
+                                </enumeratedValue>
+                                <enumeratedValue>
+                                    <name>RC2</name>
+                                    <description>Alpha value from Read Circuit 2 for output selection.</description>
+                                    <value>1</value>
+                                </enumeratedValue>
+                            </enumeratedValues>
+                        </field>
+                        <field>
+                            <name>SLBG</name>
+                            <description>Alpha blending method.</description>
+                            <bitRange>[7:7]</bitRange>
+
+                            <enumeratedValues>
+                                <enumeratedValue>
+                                    <name>CIRCUIT2</name>
+                                    <description>Blend alpha with output of Read Circuit 2.</description>
+                                    <value>0</value>
+                                </enumeratedValue>
+                                <enumeratedValue>
+                                    <name>BGCOLOR</name>
+                                    <description>Use background color.</description>
+                                    <value>0</value>
+                                </enumeratedValue>
+                            </enumeratedValues>
+                        </field>
+                        <field>
+                            <name>ALP</name>
+                            <description>Alpha blending value.</description>
+                            <bitRange>[15:8]</bitRange>
+                        </field>
+                        <field>
+                            <name>NFLD</name>
+                            <description>(Uses EXTBUF external digital in)</description>
+                            <bitRange>[16:16]</bitRange>
+                        </field>
+                        <field>
+                            <name>EXVWINS</name>
+                            <description>(Uses EXTBUF external digital in)</description>
+                            <bitRange>[41:32]</bitRange>
+                        </field>
+                        <field>
+                            <name>EXVWINE</name>
+                            <description>(Uses EXTBUF external digital in)</description>
+                            <bitRange>[51:42]</bitRange>
+                        </field>
+                        <field>
+                            <name>EVSYNCMD</name>
+                            <description>(Uses EXTBUF external digital in)</description>
+                            <bitRange>[60:52]</bitRange>
+                        </field>
```

Comment:
Found those while referencing the [gsKit](https://github.com/ps2dev/gsKit/blob/a98e74196857ff7b7a6df1779b1c9c8900220351/ee/gs/include/gsInit.h#L637). Wasn't sure what they referred to because there was no other mention but thought it worth including. Removed to keep in line with the manual.